### ISSUE TITLE
Florisboard Crashing on setup, rendering it unusable

### ISSUE THREAD

**Issue by mayur392** [florisboard/florisboard#1414]
#### Environment information
- FlorisBoard Version: 0.3.13 (56)
- Device: ONEPLUS A5000
- Android: 12 (cn=S sdk=31)

#### Features enabled
```kt
smartbar = true
suggestions = false
suggestions_clipboard = true
suggestions_next_word = true
glide = false
clipboard_internal = true
clipboard_history = false
```

#### Attached stacktrace files
<details>
<summary>1638205932794.stacktrace</summary>

```
java.lang.NullPointerException: Attempt to invoke virtual method 'void android.inputmethodservice.SoftInputWindow.setToken(android.os.IBinder)' on a null object reference
	at android.inputmethodservice.InputMethodService$InputMethodImpl.attachToken(InputMethodService.java:638)
	at android.inputmethodservice.InputMethodService$InputMethodImpl.initializeInternal(InputMethodService.java:608)
	at android.inputmethodservice.IInputMethodWrapper.executeMessage(IInputMethodWrapper.java:173)
	at com.android.internal.os.HandlerCaller$MyHandler.handleMessage(HandlerCaller.java:44)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7842)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)

```
</details>

**Comment by patrickgold** [comment]
Would it be possible that you try out the latest beta (0.3.14-beta06) to see if this crash has been fixed for you? A lot has changed since 0.3.13 and the setup screen does not look at all anymore like it did in 0.3.13.

**Comment by patrickgold** [comment]
Closing this issue as it stalled. If you can reproduce this crash on the latest beta we can reopen this issue.
